docs/serving/configuring-the-autoscaler.md

```diff
@@ -108,21 +108,43 @@ limit how many requests reach the app at a given time. Using
 enforced constraint of concurrency.
 
 `containerConcurrency` limits the amount of concurrent requests are allowed into
-the application at a given time (hard limit), and is configured in the Revision
+the application at a given time (hard limit), and is configured in the revision
 template.
 
 ```
 containerConcurrency: 0 | 1 | 2-N
 ```
 
 - A `containerConcurrency` value of `1` will guarantee that only one request is
-  handled at a time by a given instance of the Revision container.
+  handled at a time by a given instance of the revision container.
 - A value of `2` or more will limit request concurrency to that value.
 - A value of `0` means the system should decide.
 
 If there is no `/target` annotation, the autoscaler is configured as if
 `/target` == `containerConcurrency`.
 
+### Configuring scale bounds (minScale and maxScale)
+
+The `minScale` and  `maxScale` annotations can be used to configure the minimum and maximum number of pods that can serve applications.
+These annotations can be used to prevent cold starts or to help control computing costs.
+
+`minScale` and `maxScale` can be configured as follows in the revision template;
+
+```
+autoscaling.knative.dev/minScale: "2"
+autoscaling.knative.dev/maxScale: "10"
+```
+
+Using these annotations in the revision template will propagate this to `PodAutoscaler` objects. You can also manually propagate `PodAutoscaler` objects.
```

Comment:
Not sure what's meant by "manually propagate" here but we could state that `PodAutoscaler` objects are mutable and you can change these annotations after the fact on the respective `PodAutoscaler` object.